runtime/doc/quickfix.txt

```diff
@@ -1921,6 +1922,63 @@ error messages into a format that quickfix mode will understand.  See the
 start of the file about how to use it.  (This script is deprecated, see
 |compiler-perl|.)
 
-
+=============================================================================
+10. Customizing the quickfix window		*quickfix-window-function*
+
+The default format for the lines displayed in the quickfix window and location
+list window is:
+
+    <filename>|<lnum> col <n>|<text>
+
+This can be customized by setting the 'quickfixtextfunc' option to a Vim
+function.  This function will be called with a dict argument for every entry
+in the quickfix list. The dict argument will have the following fields:
```

Comment:
how do you get the text of the current "index"? presumably using `getqflist` and specifying the index in the options dict. Worth saying that here to avoid confusion. It's clear in the example below, but:

* the example is complex
* the example contains no commentary explaining what it is doing or why

Reply:
Yes. You can get the contents of a specific quickfix item using 'index' in
getqflst(). I will update the help text to explicitly state this.  I will also
add comments to the example function.